tests/tests/test_inventory.py

```diff
@@ -86,6 +119,12 @@ def test_inventory(self):
                             )
                             in attrs
                         )
+                        assert (
+                            json.loads(
+                                '{"name": "rootfs-image_swname_version", "value": "v1"}'
```

Comment:
We will need to fix this somehow (discussed in chat).

Reply:
Fixed